### PR TITLE
add cucim.core.morphology to API docs + other docstring fixes

### PR DESCRIPTION
**Pull request by grlee77**
make sure `cucim.core.morphology` module is included in the core API docs

fix typo in references section of `distance_transform_edt` docstring

setup intersphinx link to SciPy docs (used in `distance_transform_edt`)

add `sphinx.ext.mathjax` for LaTeX rendering

fix sphinx warning/typo related to `image_to_absorbance` docstring

### CODE REVIEW

File: docs/source/conf.py [hunk 1]
```diff
@@ -173,7 +174,10 @@
 
 
 # Example configuration for intersphinx: refer to the Python standard library.
-intersphinx_mapping = {'https://docs.python.org/': None}
+intersphinx_mapping = {
+    'python': ('https://docs.python.org/', None),
+    'scipy': ('https://docs.scipy.org/doc/scipy/', None),
```
Comment (jakirkham) [inline docs/source/conf.py:179]: Should we add NumPy or CuPy?

**Review by jakirkham — COMMENTED** [review]
Thanks Greg! 🙏

Had one question below

File: docs/source/conf.py (hunk 1 above)
Comment (grlee77) [reply docs/source/conf.py:181]: Yeah, I think they will be good to have even if they are not currently used. I added those two and scikit-image.

**Review by jakirkham — APPROVED** [review]

**Comment by jakirkham** [comment]
@gpucibot merge

**Comment by jakirkham** [comment]
Thanks Greg! 🙏

**Pull request merged**